src/annotator/sidebar.js

```diff
@@ -1,9 +1,11 @@
 import * as Hammer from 'hammerjs';
+import { render } from 'preact';
 
 import { addConfigFragment } from '../shared/config-fragment';
 import { sendErrorsTo } from '../shared/frame-error-capture';
 import { ListenerCollection } from '../shared/listener-collection';
 import { PortRPC } from '../shared/messaging';
+import HostToastMessages from '../sidebar/components/HostToastMessages';
```

Comment:
The `src/annotator/` module is not supposed to important components from `src/shared/` so we'll need to place the shared base components in `src/shared/components`. As for naming, some options:

- Give the more general name to the base component and a more specific name to the components which wrap it
- Add a `Base` suffix to the shared component
- Use the same name for the base and wrappers, but rename the base when importing to avoid a clash (`import { ToastMessages as BaseToastMessages } ...`).